whichNamespace.sh

```diff
@@ -10,5 +10,15 @@ else
     kubectl config use-context $CLUSTER
     echo "Running python3 namespace.py -c $CLUSTER"
     python3 namespace.py -c $CLUSTER
+    for file in namespaces/*; do
```

Comment:
This is looping through the clusters, while the loop in L8 does the same. Wouldn't this apply terraform multiple times on each run?